uber/templates/preregistration/form.html

```diff
@@ -15,83 +14,6 @@
 {% if attendee.is_dealer %}
     <span class="text-center"><h2>Dealer Application</h2></span>
 {% else %}
```

Comment:
This all moved into ``preregbase.html`` and it's pretty much the same except for the second row of options.